circle.yml

```diff
@@ -19,12 +21,16 @@ jobs:
       - image: vapor/swift:5.0
     steps:
       - checkout
-      - run: swift test -c release
+      - run:
+          name: swift test
+          command: swift test -c release --filter "RoutingKitTests.RouterPerformanceTests"
 
 workflows:
   version: 2
   tests:
     jobs:
       - linux
       - linux-release
-      - linux-performance
```

Comment:
Removing perf tests from the automatic workflow because they should be triggered manually by the bot.